bedrock/firefox/templates/firefox/messaging-experiment/privacy_by_default.html

```diff
@@ -0,0 +1,69 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+  
+  {% from "macros-protocol.html" import hero with context %}
+  
+  {% extends "firefox/welcome/base.html" %}
+  
+  {# L10n: HTML page title #}
+  {% block page_title %}{{ _('Firefox: Private By Default') }}{% endblock %}
+  
+  {% block body_class %}{{ super() }} welcome-page8{% endblock %}
+  
+  {% block content_intro %}
+    {% call hero(
+      title=_('Firefox: Private By Default'),
+      desc=_("Privacy is more than a mission. It's built right into our products. Our default privacy and security settings block malware, stop trackers, and keep companies from following you around the web."),
+      class='mzp-t-firefox mzp-t-dark',
+      include_cta=True,
+      heading_level=1
+    ) %}
+  
+    <p class="primary-cta">
+      <div>
+        <a class="mzp-c-button mzp-t-product js-open-about-protections" data-cta-text="Go to your Protections Report" data-cta-type="button">{{ _('Check out your protections') }}</a>
```

Comment:
This should probably have an href even though it triggers a JS event. Elsewhere we've linked to SUMO, just in case JS fails or is disabled. [Here's an example from an older whatsnew page.](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx70-en.html#L74)